build.yaml

```diff
@@ -151,15 +179,9 @@ common_cli:
       '
       return \${PIPESTATUS[0]}
     }
-    # fix the docker permission issue, the base image doesn't have the correct
-    # permissions/owners.
-    # This is to be tackled as part of:
-    # https://clusterhq.atlassian.net/browse/FLOC-2689
-    sudo chmod 777 /var/run/docker.sock
-
 
   cleanup: &cleanup |
-    export PATH=/usr/local/bin:\$PATH
+    export PATH=/usr/local/bin:\${PATH}
```

Comment:
wrap every $VAR, make it consistent and avoid spending another hour troubleshooting a broken groovy file due to a misslaced $\{